src/actions/EventsActions.ts

```diff
@@ -23,3 +24,26 @@ export const getListOfEventsForHome = (
       });
   });
 };
+
+export const getEventById = (id: string) => {
+  return new Promise<Event>((resolve, reject) => {
+    useQuery<getListOfEventsReturnTypes>(EVENT_BY_ID_QUERY, { id })
+      .then(({ events }) => {
+        if (!events || !events.length) {
+          Notify.create({
+            message: 'No se encontro el evento',
+            type: 'negative',
+          });
+
+          reject(null);
+          return null;
+        }
+
+        resolve(events[0]);
```

Comment:
If you use event_by_pk instead in the graphql query you won't need to get array

Reply:
Thanks, I have applied it and now looks clean!